package.json

```diff
@@ -17,23 +17,39 @@
     "build": "lerna run build",
     "clean": "lerna run clean",
     "build:tsc": "tsc -b packages/advanced-logic packages/data-access packages/data-format packages/epk-decryption packages/epk-signature packages/ethereum-storage packages/integration-test packages/multi-format packages/payment-detection packages/prototype-estimator packages/request-client.js packages/request-logic packages/request-node packages/smart-contracts packages/toolbox packages/transaction-manager packages/types packages/usage-examples packages/utils packages/web3-signature packages/currency",
-    "lint": "lerna run lint",
-    "lint-staged": "lerna run lint-staged",
+    "lint": "eslint packages/**/src/**/*.ts --quiet",
```

Comment:
`quiet` is there to ignore warnings for now, there are hundreds of them. We could disable them from config but I feel it's good to have them in the IDE